macros_doc.md

```diff
@@ -9,15 +34,40 @@ such as Excel.
 
 MkDocs (the tool we use to turn the markdown version of the specification into
 HTML pages) supports macros. In the BIDS specification document, we use these
-macros to standarize the format of items such as tables and examples.
+macros to standardize the format of items such as tables and examples.
+
+## What kind of input information are required by macros?
+
+It will depend for each macro.
+
+For some of them, all the inputs will be directly available in the markdown
+document where you have put the call to the macro.
+
+Other macros will need to be pointed to some external input (like a metadata
+term) that they are supposed to help rendering. For example, parts of the BIDS
+specification are formalized into a "schema" so that requirements in the
+specification can be automatically checked by validators. Several of the macros
+incorporate information from this schema to assure consistency.
 
-Increasingly, parts of the BIDS specification are being formalized into a
-"schema" so that requirements in the specification can be automatically checked
-by validators. Several of the macros incorporate information from the schema to
-assure consistency.
+## What macros are available? What can we use macros for?
+
+All the macros we use are in listed in this
+[python file](https://github.com/bids-standard/bids-specification/blob/master/tools/mkdocs_macros_bids/macros.py).
+
+| Name                    | Purpose                                                                                | Uses schema | Link to example of use |
+| ----------------------- | -------------------------------------------------------------------------------------- | ----------- | ---------------------- |
```

Comment:
`make_filetree_example` doesn't use the schema does it?  Shouldn't it say No in the third column

The third column should have consistent capitalization.

Also, should the macros be in alphabetical order in the table?

Reply:
Correct. Fixing this.

> `make_filetree_example` doesn't use the schema does it? Shouldn't it say No in the third column

Correct. Fixing this.

Reply:
> The third column should have consistent capitalization.
> 
> Also, should the macros be in alphabetical order in the table?

Done and done.